src/pages/policy/input/ParameterEditor.jsx

```diff
@@ -30,6 +31,17 @@ export default function ParameterEditor(props) {
     startDate,
   );
 
+  useEffect(() => {
+    const [parameterStartDate, parameterEndDate] = policy.reform.data[
+      parameterName
+    ]
+      ? Object.keys(policy.reform.data[parameterName])[0].split(".")
+      : [defaultStartDate, defaultEndDate];
+    setStartDate(parameterStartDate);
+    setEndDate(parameterEndDate);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Ideally, we should really avoid manually disabling `exhaustive-deps`. I know there are points in the code where it is utilized, don't get me wrong, but those are legacy and we're moving away from them. I believe that if you rewrote this to only actually update state when parameterName exists within `policy.reform.data`, you might avoid whatever issue you ran into that caused you to add the `eslint-disable-next-line` call.